connectors/src/lib/api/config.ts

```diff
@@ -0,0 +1,11 @@
+import { EnvironmentConfig } from "@dust-tt/types";
+
+export const apiConfig = {
+  getDustAPIConfig: (): { url: string; nodeEnv: string } => {
+    return {
+      // Dust production API URL is hardcoded for now.
+      url: "https://dust.tt",
```

Comment:
We use to have `DUST_PROD_API` why don't use it?

Reply:
It's not defined neither in prod nor in development

Reply:
I see we were setting a default value to an env variable when destructuring. Such a bad pattern!